services/storage/object/controls.yaml

```diff
@@ -10,14 +10,13 @@ controls:
   - id: CCC.ObjStor.C01
     title: Prevent Requests to Buckets or Objects with Untrusted KMS Keys
     objective: |
-      Prevent any requests to object storage buckets or objects using untrusted
-      KMS keys to protect against unauthorized data encryption that can impact
-      data availability and integrity.
+      Prevent any requests to object storage buckets or objects using 
+      untrusted KMS keys to protect against unauthorized data encryption 
+      that can impact data availability and integrity.
     control_family: Data
     threats:
       - CCC.TH01
       - CCC.TH02
```

Comment:
Should we add comments here like we do with common features etc?

Reply:
Yeah, that's a good idea. I'll aim for another PR with that, but it shouldn't be a blocker for cutting the release candidate.